### PR TITLE
Modify rule S127: promote C# to SonarWay

### PR DESCRIPTION
**Pull request by cristian-ambrosini-sonarsource**
S127 CSharp:
- RSPEC cleanup
- Promote rule to `SonarWay` profile

### CODE REVIEW

File: rules/S127/rspecator.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+ifdef::env-github,rspecator-view[]
```
Comment (sebastien-marichal) [inline rules/S127/rspecator.adoc:1]: Nitpick:
```suggestion
ifdef::env-github,rspecator-view,env-vscode[]
```

File: rules/S127/rspecator.adoc [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+include::parameters.adoc[]
+
+'''
+== Comments And Links
+(visible only on this page)
+
+include::comments-and-links.adoc[]
+
+endif::env-github,rspecator-view[]
```
Comment (sebastien-marichal) [inline rules/S127/rspecator.adoc:17]: With previous comment:
```suggestion
endif::env-github,rspecator-view,env-vscode[]
```

**Review by sebastien-marichal — APPROVED** [review]
LGTM!
Left one little nitpick comment

File: rules/S127/rspecator.adoc (hunk 2 above)
Comment (cristian-ambrosini-sonarsource) [reply rules/S127/rspecator.adoc]: It's failing the CI with `,env-vscode[]`, I'll revert

**Pull request merged**